src/client/automation/playback/type/index.js

```diff
@@ -63,6 +63,11 @@ export default class TypeAutomation {
         };
 
         this.nativeAutomationInput = dispatchNativeAutomationEventFn ? new NativeAutomationInput(dispatchNativeAutomationEventFn) : null;
+
+        // NOTE: we disable the `paste` option for the `nativeAutomation` mode since
+        // is a synthetic option that don't have equivalent for native user action
+        if (this.nativeAutomationInput)
```

Comment:
I propose using another way: if there is the `paste` option, don't use the `nativeAutomation` approach.